src/platform/forms/tests/save-in-progress/SaveInProgressDevModal.unit.spec.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM /* , { act } */ from 'react-dom';
 import { expect } from 'chai';
 import ReactTestUtils from 'react-dom/test-utils';
-
+import '@department-of-veterans-affairs/component-library/i18n-setup';
```

Comment:
Sanity check; were you able to get the Check-In app running locally with the import here and see that the web components were displaying text as expected? It might be worth tossing a couple of screenshots of that into the description of this PR if so.